_posts/2019-03-17-three-years-of-react-native.markdown

```diff
@@ -1,6 +1,6 @@
 ---
 layout: epic
-title: "React Native, 3 years later"
+title: "React Native at Artsy, 3 years later"
```

Comment:
This post isn’t a reflection on RN itself, but how it changed Artsy. Open to better suggestions.